README.md

```diff
@@ -200,11 +200,13 @@ These variables should be put in the `.env` file in the root folder of this proj
 - `CLIENT_SECRET`: OAuth Client's secret (optional, authentication is needed for examples that use Processing API)
 
 - `INSTANCE_ID`: id of the configuration instance that will be used in examples
+- `S1GRDIW_LAYER_ID`: id of the Sentinel-1 GRD IW layer from that instance
+- `S1GRDEW_LAYER_ID`: id of the Sentinel-1 GRD EW layer from that instance
 - `S2L2A_LAYER_ID`: id of the Sentinel-2 L2A layer from that instance
-- `S1GRD_LAYER_ID`: id of the Sentinel-1 GRD layer from that instance
+- ... (see `.env.example` for full list)
 
 Instance can be created with the [**Configurator**](https://apps.sentinel-hub.com/dashboard/#/configurations) on the **Sentinel Hub Dashboard**.
-It should contain at least one Sentinel-2 L2A layer and one Sentinel-1 GRD layer, whose layer IDs should be the same as set in `S2L2A_LAYER_ID` and `S1GRD_LAYER_ID` env vars respectively.
+It should contain at least one Sentinel-2 L2A layer and one Sentinel-1 GRD layer, whose layer IDs should be the same as set in `S2L2A_LAYER_ID` and `S1GRDIW_LAYER_ID` env vars respectively.
```

Comment:
I never particularly liked that sentence and now with the storybook examples for S1 GRD (IW & EW) and S3 (OLCI & SLSTR), it's also outdated.

I was thinking we can maybe change it to something similar to "It should contain the layers that are mentioned in the list above."

Reply:
Agreed - I fixed it a bit.

Reply:
Nice. I'm nitpicking, now, but maybe change `stories` with `examples`. That way it would cover both node examples and storybook stories.

Reply:
Done. :)

Reply:
Nice :) Resolving.